scripts/mod_loader/modui/pilot_arrange.lua

```diff
@@ -111,13 +111,30 @@ local function createUi()
 			end
 		end
 		
+		local function addHangarButton(i)
+			local col = (i - 1) % portraitsPerRow
+			local row = math.floor((i - 1) / portraitsPerRow)
+			
+			local surface = sdlext.surface("resources/mods/ui/pilot_arrange_hangar.png")
```

Comment:
This creates a surface each time a button is created, which is very inefficient (particularly because this is an actual SDL object). It's better to move this to the top of the file next to `MAX_PILOTS`, so that the surface is only created once, and then re-used by UI objects.